utilities/ImpactUtilities.groovy

```diff
@@ -386,6 +392,56 @@ def matches(String file, List<PathMatcher> pathMatchers) {
 	return result
 }
 
+/**
+ *  shouldCalculateImpacts
+ *  
+ *  Method to calculate if impact analysis should be performed for a changedFile in an impactBuild scenario
+ *   returns a boolean - default true
+ */
+def boolean shouldCalculateImpacts(String changedFile){
+	// retrieve Pathmaters from property and check
+	List<PathMatcher> nonImpactingFiles = createPathMatcherPattern(props.skipImpactCalculationList)
+	onskipImpactCalculationList = matches(changedFile, nonImpactingFiles)
+
+	// return false if changedFile found in skipImpactCalculationList
+	if (onskipImpactCalculationList) return false
+	return true //default
+
+	//	// Alternate implementation using a property file
+	//	// file named <*buildfile*_prop.properties> in the same folder
+	//	//	 containing a property <impact_changes=true|false> to indicate to skip impact analysis
+	//	propertyFileName = changedFile.replaceAll(".cpy","_prop.properties")
+	//	File propertyFile = new File("${props.workspace}/${propertyFileName}")
+	//	if (propertyFile.exists())
+	//	{
+	//		Properties impactProperties = new Properties()
+	//		propertyFile.withInputStream { impactProperties.load(it) }
+	//		impactChanges = impactProperties.get('impact_changes')
+	//		if (impactChanges != null){
+	//			if (impactChanges.toBoolean()==false) return false
+	//		}
+	//	}
+	//	return true // default
+}
```

Comment:
Can you remove this code in comments ?

Reply:
The initial idea was to have a choice, but you are right. We should move this sample snippet to https://github.com/ibm/dbb